workers/unity/Packages/com.improbable.gdk.tools/RedirectedProcess.cs

```diff
@@ -9,30 +10,136 @@
 namespace Improbable.Gdk.Tools
 {
     /// <summary>
-    ///     Runs a windowless process with its stdout/stderr redirected to the Unity console as a single debug print at the
-    ///     end.
+    ///     Runs a windowless process and waits for it to return.
     /// </summary>
-    public class RedirectedProcess
+    public static class RedirectedProcess
```

Comment:
This feels like we should rewrite this into a builder like API.

I.e. -

```
	var exitCode = RedirectedProcess
		.Command(...)
		.WithArgs(...)
		.InWorkingDirectory(...)
		.CollectOutput(out var output)
```

Reply:
Good point, let me try that!